README.md

```diff
@@ -1,25 +1,57 @@
-# Screen Latency Testing
+# Screen Latency Automated Testing (SLATE)
 
 **This is not an officially supported Google product**
 
-This is an android app that uses OCR to detect text appearing in the field of
-view of the camera (typically pointed at a screen or laptop display).
+This is a 2 part software that helps to measure typing latency on display devices.
 
-It attempts to provide a high accuracy as to when the text changes by:
+In a more elaborate sense, it measures how much time it takes a key you typed to appear on your screen. As the latency can vary with the use of a different keyboard, this software carries the measurement only from the point operating system gets to know a key-press input is received, as that measurement will be keyboard independent.
 
-1. Using high camera frame rates if available
+## Install instructions
+1. Clone the repository or download it from the source.
 
-2. Using synchronization to work out time differences between the phone and  the
-   server controlling the test. 
+   `git clone https://github.com/googleinterns/screen-latency-testing.git`
 
-The motivating use case is measuring the delay between key events and
-charactersappearing on the display.  It may also be useful for additional
-testing scenarios. 
+2. Import the `Camera2SlowMotion` android app in Android Studio.
 
-## Building
+3. Install the app on your smartphone from Android Studio. (App will not run without step 5)
 
-TODO 
+4. Build the server script.
 
-## Using
+   `go build Server/serverSlate.go`
+   
+5. Connect your smartphone to your laptop/pc/testing-device using a USB cable and run the server script on your laptop/pc/testing-device.
```

Comment:
Maybe you should mention that phone needs to be in dev mode with usb debugging enabled somewhere in these instructions.

Reply:
@mwarton Added as point 5.